core/src/main/scala/io/finch/EndpointModule.scala

```diff
@@ -289,6 +290,22 @@ trait EndpointModule[F[_]] {
   def asyncBody(implicit F: Effect[F]): Endpoint[F, AsyncStream[Buf]] =
     Endpoint.asyncBody[F]
 
+  /**
+    * An alias for [[Endpoint.streamBody]]
+    */
+  def streamBody[S[_[_],_], A, CT <: String](implicit
```

Comment:
I like this a lot.

My only question would be, we should figure out if we can make this work with `AsyncStream[Buf]` as a stream or a plain `Reader[Buf]` as a stream.

Reply:
Check https://github.com/finagle/finch/pull/1042/commits/b03b88ad2339cebc1a106bb4f203d64895fde56e

It's possible using type alias `AsyncStreamF`. Also, I don't see much of a value to add straight `Reader[Buf]` support as one could always use `root` endpoint to extract it from request.

Reply:
`AsyncStream` is going to be deprecated